app/internal/hebrew_date_view.py

```diff
@@ -0,0 +1,72 @@
+from datetime import datetime
+
+from hebrew_numbers import int_to_gematria
+from pyluach import dates, hebrewcal
+
+
+EN_TO_HEB_MONTH_NAMES = {
+    'Tishrei': 'תשרי',
+    'Cheshvan': 'חשוון',
+    'Kislev': 'כסלו',
+    'Teves': 'טבת',
+    'Shvat': 'שבט',
+    'Adar': 'אדר',
+    'Adar Rishon': "אדר(א')",
+    'Adar Sheni': "אדר(ב')",
+    'Nissan': 'ניסן',
+    'Iyar': 'אייר',
+    'Sivan': 'סיוון',
+    'Tamuz': 'תמוז',
+    'Av': 'אב',
+    'Elul': 'אלול',
+}
+
+
+def get_month_name_by_num(date: datetime) -> str:
+    """Returns the Hebrew name date for the specific day.
+
+        Args:
+            date: The requested date.
+
+        Returns:
+            A Hebrew name date.
+        """
+
```

Comment:
Remove extra line after docstrings in all functions

Reply:
OK